### DISCUSSION TITLE
Share links, podcasts, videos, books...to add to the website

### DISCUSSION THREAD

**Discussion by ObjectifMoon21** [sovenor/bitcoin-rocks#33] [Ideas]
Share educational contents here that could be added to the website. Specify which category or page.

Bitcoin vs CBDCs: 
https://cbdctracker.hrf.org/home

Energy: 
https://youtube.com/shorts/6x_nfuNigI8?si=VRuSYls6ADjbl66G
https://www.youtube.com/live/xFEVbzX8M0w?si=7P26HbsidrEboT2K

Global bitcoin stats all-in-one website: 
https://timechainstats.com/
https://mempool.space/

**Comment by sovenor** [comment]
These are really great recommendations, thank you @ObjectifMoon21! I will review this and add at least some of them ASAP.

**Comment by ObjectifMoon21** [comment]
Today when I placed some stickers, someone asked me about bitcoin and how to buy it. On the site there's a link to strike. I think it would be nice to add other no-kyc options.

Buy btc with Euro sepa or credit card, Light KYC, self-custody wallet built-in on-chain & Lightning in beta: [Relai app](https://relai.app/)

No-KYC, P2P with many payments options, self-custody on-chain wallet built-in, users rating: [Peach bitcoin](https://peachbitcoin.com/)

No-KYC, P2P, Lightning only with invoice, Tor integration, the most anonymous online p2p market, you can choose to keep your user id for rating or create a new user for each transactions: [RoboSats](https://learn.robosats.com/)
 [RoboSats on GitHub](https://github.com/RoboSats/robosats)

**Comment by sovenor** [comment]
> Share educational contents here that could be added to the website. Specify which category or page.
> 
> Bitcoin vs CBDCs: https://cbdctracker.hrf.org/home
> 
> Energy: https://youtube.com/shorts/6x_nfuNigI8?si=VRuSYls6ADjbl66G https://www.youtube.com/live/xFEVbzX8M0w?si=7P26HbsidrEboT2K
> 
> Global bitcoin stats all-in-one website: https://timechainstats.com/ https://mempool.space/

I added the CBDC tracker and one of the Energy links. I'm not sure of a good place to put the bitcoin stats right now. Thanks for these suggestions!
